config/checkstyle/checkstyle.xml

```diff
@@ -212,7 +212,7 @@
       <property name="option" value="NL"/>
       <property name="tokens"
                 value="BAND, BOR, BSR, BXOR, DIV, EQUAL, GE, GT, LAND, LE, LITERAL_INSTANCEOF, LOR,
-                    LT, MINUS, MOD, NOT_EQUAL, PLUS, QUESTION, SL, SR, STAR, METHOD_REF "/>
+                    LT, MINUS, MOD, NOT_EQUAL, QUESTION, SL, SR, STAR, METHOD_REF "/>
```

Comment:
Changing this checkstyle here was my initiative, but I want it to be clear that we shouldn't.

Also: I'm doing it as a TL. As devs, DO NOT modify this file of your own initiative, please.